src/components/Navbar/Navbar.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
 import logo from '../Assets/space-logo.png';
 import './Navbar.css';
 
```

Comment:
It is better to use NavLink as it is a more modernized way to implement Navbar feature.